java/arcs/core/storage/handle/SingletonImpl.kt

```diff
@@ -48,8 +48,9 @@ class SingletonImpl<T : Referencable>(
      * did not apply fully. Fetch the latest value and retry.
      * */
     suspend fun store(entity: T): Boolean {
-        versionMap.increment()
-        return storageProxy.applyOp(CrdtSingleton.Operation.Update(name, versionMap, entity))
+        return storageProxy.applyOp(
+            CrdtSingleton.Operation.Update(
+                name, versionMap().increment(), entity))
```

Comment:
And here

```kotlin
return storageProxy.applyOp(
    CrdtSingleton.Operation.Update(
        name,
        versionMap().increment(),
        entity
    )
)
```

Pro-tip, you can do this with Alt+Enter while your cursor is on one of the arguments, and choose "Place arguments on separate lines"

Reply:
Thanks, all done.